src/views/properties.js

```diff
@@ -89,7 +89,7 @@ export class Properties extends Component {
                                 <Link className="button is-rounded" to="/add/property">+ ADD NEW</Link>
                             </div>
                             <div className="search-section">
-                                <input></input>
+                              <input class="input search is-rounded" placeholder="Search properties by name, address, or property manager"></input>
```

Comment:
the "class" prop here will have to be "className" instead for React to pick it up

Reply:
Another sloppy copy/paste from Bulma, but mostly using this as a dummy input until we build the real search component.